File: Java/Lista_04(Vetores)/Aprovados.java
import java.util.Locale;
import java.util.Scanner;

public class Aprovados {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner s = new Scanner(System.in);

        System.out.print("Quantos alunos serao digitados? ");
        int n = s.nextInt();

        String nome[] = new String[n];
        double nota1[] = new double[n];
        double nota2[] = new double[n];

        for (int i = 0; i < nome.length; i++) {
            System.out.println("Digite nome, primeira e segunda nota do " + (i + 1) + "o aluno:");
            s.nextLine();
            nome[i] = s.nextLine();
            nota1[i] = s.nextDouble();
            nota2[i] = s.nextDouble();
        }

        double media[] = new double[n];

        for (int i = 0; i < nome.length; i++) {
            media[i] = (nota1[i] + nota2[i]) / 2;
        }

        System.out.println("Alunos aprovados:");

        for (int i = 0; i < nome.length; i++) {
            if (media[i] >= 6.0) {
                Locale.setDefault(Locale.US);
                System.out.println(nome[i]);
            }
        }

        s.close();

    }
}

File: Python/Lista_04(Vetores)/Aprovados.py
n = int(input("Quantos alunos serao digitados? "))

nome = [0 for x in range(n)]
nota1 = [0 for x in range(n)]
nota2 = [0 for x in range(n)]
media = [0 for x in range(n)]

for i in range(0,n):
    print(f"Digite nome, primeira e segunda nota do {i+1}o aluno:")
    nome[i] = input()
    nota1[i] = float(input())
    nota2[i] = float(input())

for i in range(0,n):
    media[i] = (nota1[i]+nota2[i])/2

print("Alunos aprovados:")
for i in range(0,n):
    if media[i]>=6.0:
        print(nome[i])